tox.ini

```diff
@@ -2,6 +2,7 @@
 envlist =
     py{39,38,37,36,py3}
     style
+    typing
```

Comment:
To actually run this in CI you need to add it to the GitHub actions workflow as well: https://github.com/pallets/flask/blob/master/.github/workflows/tests.yaml#L26

Reply:
Ta